docs/oauth2-oidc/get-started/social-sign-in.mdx

```diff
@@ -0,0 +1,206 @@
+---
+id: social-sign-in
+title: Configure social sign-in with Ory Identities
+sidebar_label: Social sign-in with Ory Identities
```

Comment:
This is a nice gimmick thing that can be set up, but I'm wondering if it should live in "Get Started" or in some advanced guides, as I can't really imagine a use case where this makes sense?

Reply:
maybe for a "login with $company" use case as described in this blogpost https://www.ory.sh/blog/run-sso-oauth2-enterprise-server-open-source/ ?